spec.html

```diff
@@ -28051,8 +28315,24 @@ <h2>Syntax</h2>
 
         ImportedBinding :
           BindingIdentifier[~Yield, +Await]
+
+        WithClause :
+          `with` `{` `}`
+          `with` `{` WithEntries `,`? `}`
+
+        WithEntries :
+          AttributeKey `:` StringLiteral
+          AttributeKey `:` StringLiteral `,` WithEntries
+
+        AttributeKey :
```

Comment:
Why not use `LiteralPropertyName` here?

Reply:
`LiteralPropertyName` also allows numeric literals -- it has never been discussed and there is no strong reason to disallow it, but there isn't a strong reason to _allow_ it either.

(btw, this comment uncovered a V8 bug! https://bugs.chromium.org/p/v8/issues/detail?id=14069)

Reply:
I think they should be allowed. It's an inconsistency with object initialisers for no reason.